src/main/java/org/folio/dew/batch/bulkedit/jobs/BulkEditItemProcessor.java

```diff
@@ -109,6 +111,13 @@ public ItemFormat process(Item item) {
     return itemFormat.withOriginal(item);
   }
 
+  private String generateHoldingsData(Item item) {
+    var effectiveLocationName = isEmpty(item.getEffectiveLocation()) ? EMPTY : item.getEffectiveLocation().getName();
+    var effectiveCallNumber = effectiveCallNumberComponentsToString(item.getEffectiveCallNumberComponents());
```

Comment:
Such approach may be used only if item's call number and location (both permanent and temporary) are absent. Otherwise, you need to get holdings record and retrieve data from there.

Reply:
done